### PR TITLE
Test backwards compatibility of Accessory._set_services

### PR DESCRIPTION
**Pull request by ikalchev**

### CODE REVIEW

File: tests/test_accessory.py [hunk 1]
```diff
@@ -9,39 +9,50 @@
...
         char.set_value(25, should_notify=True)
 
+    def test_set_services_compatible(self):
+        """Test that _set_services still works and has access to the info services"""
+        class A(accessory.Accessory):
+            def _set_services(self):
+                super()._set_services()
+                s = loader.get_serv_loader().get_service("TemperatureSensor")
+                self.add_service(s)
+                assert(self.get_service("AccessoryInformation") is not None)
```
Comment (cdce8p) [inline tests/test_accessory.py]: `assert self.get_service("AccessoryInformation") is not None` should work

File: tests/test_accessory.py [hunk 2]
```diff
@@ -9,39 +9,50 @@
...
+        a = A("Test Accessory")
+        assert(a.get_service("TemperatureSensor") is not None)
```
Comment (cdce8p) [inline tests/test_accessory.py]: Same here

Comment (ikalchev) [reply tests/test_accessory.py]: Thanks! Writing cpp during the day and python at night/weekend really messes up your coding habits

**Comment by cdce8p** [comment]
Looks good to me :+1:

**Pull request merged**